CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+Contributing to Okta Open Source Repos
+======================================
+
+Sign the CLA
+------------
+
+you _may_ need to [sign the CLA](https://developer.okta.com/cla/), read it before getting started. Common questions/answers are also listed on the CLA page.
```

Comment:
**Y**ou (capitalization)

Reply:
Also do we want to clarify what _may_ means?  Like in what scenarios?  Feels a bit vague.

Reply:
It is explained in the CLA page.

Reply:
Not really, unless I'm missing something.  The CLA page explains what a CLA is and why it's important, but it makes it sound like it's always required to contribute.  I'm specifically curious why this line says _may_ (emphasis yours), which implies there are some scenarios where it's required and some where it's not.  Either I'm missing something, or it might be better to say `You will need to sign...`

``
This CLA allows us to merge in code (or build upon suggested code) in our sample apps and SDKs, giving us the ability to utilize that code in the course of our business.
```

Reply:
Have you checked this one: https://developer.okta.com/cla/?

In there, it describes that there are obvious fixes that may not require to sign a CLA. Then, it gives a couple of examples of what is considered an obvious fix.